el-tck/pom.xml

```diff
@@ -27,7 +27,7 @@
     </parent>
 
     <artifactId>jakarta-expression-language-tck</artifactId>
-    <version>6.0.1</version>
+    <version>11.0.0-SNAPSHOT</version>
```

Comment:
el-tck module (artifactId: jakarta-expression-language-tck) is the standalone TCK which does not run in vehicles and is not part of platform TCK. The latest version released was 6.0.1 for the specification release.

el module is the right one which is refactored for platform TCK , and run in vehicles.

Reply:
Ok, this has been changed.